### PR TITLE
Update to 2.9.5

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,19 +7,13 @@ package:
...
-    - fix-attribute-decoding-during-xml-schema-validation.patch
+  sha256: 7aaf3e1f03317231864f833eea25a30721cc7b434884e089cc3a021b6d58f60c
 
 build:
-  number: 4
+  number: 0
   skip: True  # [win and [py>35]
   features:
     - vc9  # [win and py27]
-    - vc10  # [win and py34]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Please don't remove these. `conda-forge` is not building py34 and this has no bearing here, but we have many people who fetch our recipes and re-build them.

**Comment by jschueller** [comment]
I wonder why circleci builds does not build anything.

File: recipe/meta.yaml (hunk 1 above)
Comment (jschueller) [reply recipe/meta.yaml]: But what if I do want to make their life more painful so as they give up and stop maintaining deprecated versions of Python and make them contribute :?

Comment (ocefpaf) [reply recipe/meta.yaml]: Please don't ;-p

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,19 +7,13 @@ package:
...
-    # Bug 767063 - Fix build of libxml2 2.9.4 (git master) on pre-C99 compilers.
-    - pre-c99.patch
-    # https://bugzilla.gnome.org/show_bug.cgi?id=766834
-    - fix-attribute-decoding-during-xml-schema-validation.patch
+  sha256: 7aaf3e1f03317231864f833eea25a30721cc7b434884e089cc3a021b6d58f60c
 
 build:
-  number: 4
+  number: 0
   skip: True  # [win and [py>35]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: The selector here is broken, I guess that is what is wrong with CircleCI.

Comment (jschueller) [reply recipe/meta.yaml]: Right, thanks

File: recipe/meta.yaml [hunk 3]
```diff
@@ -34,15 +28,13 @@ requirements:
     - zlib 1.2.*
     - xz 5.2.*  # [not win]
     - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Please add these back.

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipe/meta.yaml]: This could have messed up the rendering as well. So may need a re-render.

**Pull request merged**